### PR TITLE
Fix exporting PNG or SVG files with multiple pages

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #18369 

🤦

### CODE REVIEW

File: src/project/internal/exportprojectscenario.cpp [hunk 1]
```diff
@@ -266,14 +266,13 @@ bool ExportProjectScenario::guessIsCreatingOnlyOneFile(const notation::INotation
...
-                = std::find_if(potentialExcerpts.cbegin(), potentialExcerpts.cend(),
-                               [notation = notations.front()](const IExcerptNotationPtr& excerpt) {
+            auto it = std::find_if(potentialExcerpts.cbegin(), potentialExcerpts.cend(),
+                                   [notation = notations.front()](const IExcerptNotationPtr& excerpt) {
                     return excerpt->notation() == notation;
                 });
 
             if (it == potentialExcerpts.cend()) {
-                return notations.front()->elements()->pages().size();
+                return notations.front()->elements()->pages().size() == 1;
```
Comment (Jojo-Schmitz) [inline src/project/internal/exportprojectscenario.cpp:275]: I wonder why this didn't lead to a compiler warning, returning a `size_t` where a `bool` is wanted

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
@cbjeukendrup It looks good now, but I found a case where it still happens: if to **switch score to Continuous view** (horizontal/vertical)


https://github.com/musescore/MuseScore/assets/90187801/039bd9ed-1ff1-4591-af4d-85c844349531

**Comment by Jojo-Schmitz** [comment]
Shouldn't the export always temporarily switch to page view? IIRC that is what Mu3 did

**Comment by cbjeukendrup** [comment]
@DmitryArefiev That should be better now!

**Comment by DmitryArefiev** [comment]
@cbjeukendrup Yes, looks good now!

Tested #18369 on Win10, Mac13, LinuxUbuntu 22.04 - **FIXED**

**Pull request merged**